sia/wallet.go

```diff
@@ -52,6 +53,6 @@ func (c *Core) CoinAddress() (consensus.CoinAddress, error) {
 }
 
 // Returns a []byte that's supposed to be json of some struct.
-func (c *Core) WalletInfo() ([]byte, error) {
+func (c *Core) WalletInfo() (components.WalletStatus, error) {
```

Comment:
either `WalletInfo()` should return a `WalletInfo`, or `WalletStatus()` should return a `WalletStatus`.